src/CoreFx.Private.TestUtilities/src/CoreFx.Private.TestUtilities.csproj

```diff
@@ -29,6 +29,9 @@
     <Compile Include="$(CommonPath)\System\PasteArguments.cs">
       <Link>Common\System\PasteArguments.cs</Link>
     </Compile>
+    <Compile Condition="'$(TargetGroup)' == 'uap'" Include="$(CommonTestPath)\System\Threading\ThreadTestHelpers.cs">
```

Comment:
I don't understand one thing, how are test projects (for instance System.Threading\tests) getting access to RemoteExecutorTestBase without referencing the project? It has a project reference only to RemoteExecutorConsoleApp.csproj which in turn also does not reference the CoreFX.Private.TestUtilities project. What am I missing?